packages/appstore/src/components/modals/wallets-upgrade-modal/wallets-upgrade-modal.tsx

```diff
@@ -1,64 +1,72 @@
 import React from 'react';
 import { useWalletMigration } from '@deriv/hooks';
-import { Button, Icon, Text, Modal } from '@deriv/components';
+import { Button, Text, Modal } from '@deriv/components';
 import { observer, useStore } from '@deriv/stores';
 import { localize, Localize } from '@deriv/translations';
 import './wallets-upgrade-modal.scss';
-
-const MODAL_TRANSITION_TIMEOUT_MS = 250; // matching the default one
+import classNames from 'classnames';
+import { getUrlBase } from '@deriv/shared';
 
 const WalletsUpgradeModal = observer(() => {
```

Comment:
small question: shouldn't the unit test for this component be updated too?

Reply:
Already updated 🫡